### PR TITLE
make config work in Windows

### PR DESCRIPTION
**Pull request by PeninaSchuss**

### CODE REVIEW

File: conf/unique_conf/default_conf/developer.conf [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (pepgid) [inline conf/unique_conf/default_conf/developer.conf:1]: This file should be ignored, it contains secrets (read about .gitignore) add this regex at unique_conf dir
and delete this file from history

File: example_conf/developer.example.conf [hunk 2]
```diff
@@ -0,0 +1,42 @@
+
+explain = "copy the directory and change the file names to developer.conf"
```
Comment (pepgid) [inline example_conf/developer.example.conf:2]: rename this file to developer.conf (The exampleness of it is evident from the parent directory)

File: src/pep_data/test/t_conf_windows.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```
Comment (pepgid) [inline src/pep_data/test/t_conf_windows.py:1]: Why do you need this file? it has the same code as the adjacent t_project_conf.py. script, if you do pull request erase it from git history

**Review by pepgid — CHANGES_REQUESTED** [review]

File: src/pep_data/test/t_conf_windows.py (hunk 3 above)
Comment (PeninaSchuss) [reply src/pep_data/test/t_conf_windows.py:1]: its not exactly the same.
you gave me this file in slack few weeks ago.
do you want me to delete that?

**Pull request closed without merging**